run_model

```diff
@@ -124,11 +126,16 @@ if __name__=="__main__":
         # step 2: delphes
         log_fname = "log_delphes.log"
         delphes_exe = "DelphesHepMC2"
-        root_fname = "events.root"
+        
         if os.path.exists(root_fname):
             if args["common"].verbose: print(f'removing old {root_fname}')
             os.remove(root_fname)
         delphes_cmd = f'gunzip -c {hepmc_fname_gz} | {delphes_exe} {delphes_fname} {root_fname}'
         if not args["common"].quiet: print(f'Running Delphes ({log_fname})')
         run_cmd(delphes_cmd, log_fname, args["common"].verbose, shell=True)
         if args["common"].verbose: print(f'wrote {root_fname}')
+        
+    if 'hist' in args["common"].steps:
+        
+        Histogram.histogram(root_fname, svjHelper.darkHadronIDs, svjHelper.stableIDs)
```

Comment:
1. need to pass `helper` (the instance of the class, not the class itself)
2. 2. I think it would be better to pass in the entire helper object and then access these variables inside the `histogram()` function, in case we want to use the helper for other things later in that function

Reply:
Ah I actually was confused should I pass the instance of the class or the instance variables. So can it be done the following way?
from svjHelper import svjHelper as helper
or
from svjHelper import svjHelper 
helper = svjHelper

Reply:
`svjHelper` is the name of the class, and `helper` is an instance of that class. `helper` is already created inside run_model (https://github.com/cms-svj/model_building/blob/8a3d677970d663896c86c8489f947fc5cf90a6b6/run_model#L69), so you just have to pass it in.

Reply:
oh right! I didn't notice that. Sorry. Fixing it rn.